MapleServer2/App.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<configSections>
+		<!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
+		<section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+	</configSections>
+	<startup>
+		<supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
+	</startup>
+	<connectionStrings>
+		<add name="Maplestory2DBEntities" connectionString="metadata=res://*/MS2DBModel.csdl|res://*/MS2DBModel.ssdl|res://*/MS2DBModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=(localdb)\MSSQLLocalDB;initial catalog=Maplestory2DB;integrated security=True;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" />
```

Comment:
Why do we need a configuration file in `MapleServer2`? And why are there two connection strings in this PR?

Reply:
This is needed in order for files from mapleserver2 to run the query methods from databasehandler solution.

Reply:
Went ahead and deleted the one from databasehandler solution